trillian/ctfe/serialize_test.go

```diff
@@ -177,7 +178,7 @@ func TestSignV1TreeHead(t *testing.T) {
 
 	// But changing the contents does change the signature.
 	sth.TreeSize = 11
-	if err := signV1TreeHead(signer, &sth); err != nil {
+	if err := c.signV1TreeHead(signer, &sth); err != nil {
```

Comment:
Could make this test stronger by calling it in a loop several times for each signature and making sure it hasn't changed unless the input changes. Doesn't really test the cache update at the moment.